config/.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+require: rubocop-rails
+require: rubocop-rspec
```

Comment:
Osobiście uważam, że ładniej wygląda w ten sposób:
```
require:
  - rubocop-rails
  - rubocop-rspec
```
ale to tylko moje zdanie 😄
Natomiast na koniec pliku [powinno się dodawać newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) ;)